shared/output/cloud/showcase/lib/google/showcase/v1alpha3/identity/credentials.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "googleauth"
+
+module Google
+  module Showcase
+    module V1alpha3
+      module Identity
+        class Credentials < Google::Auth::Credentials
+          SCOPE = [
+            "https://www.googleapis.com/auth/cloud-platform"
+          ].freeze
+          PATH_ENV_VARS = %w[IDENTITY_CREDENTIALS
```

Comment:
Is this supposed to be {PRODUCT}_CREDENTIALS?

Reply:
Yes, but since there was no `env_prefix` configured for showcase, it is using the service name, which is "Identity". If we configured an `env_prefix` value in the configuration file it would use that.

Reply:
Oh, wait! This is wrong. It isn't using the prefix. Fixed.

Reply:
Thanks!